src/components/Budget/BudgetLineitem/__test__/BudgetLineItem.test.tsx

```diff
@@ -208,25 +208,30 @@ beforeEach(() => {
 	});
 });
 
-const inputIds = budgetLineItemInputFields;
+const budgetLineItemInputIds = budgetLineItemInputFields;
 
 describe("Budget Line Item Dialog tests", () => {
 	test("Mock response", async () => {
-		// 	for (let i = 0; i < inputIds.length; i++) {
-		// 		let fieldName = (await dialog.findByTestId(inputIds[i].id)) as HTMLInputElement;
-		// 		let value = intialFormValue[inputIds[i].key];
-		// 		await act(async () => {
-		// 			await fireEvent.change(fieldName, { target: { value } });
-		// 		});
-		// 		await expect(fieldName.value).toBe(value);
-		// 	}
-		// 	await act(async () => {
-		// 		let saveButton = await dialog.getByTestId("createSaveButton");
-		// 		expect(saveButton).toBeEnabled();
-		// 		fireEvent.click(saveButton);
-		// 		await wait();
-		// 	});
-		// 	await new Promise((resolve) => setTimeout(resolve, 1000));
-		// 	expect(creationOccured).toBe(true);
+		await new Promise((resolve) => setTimeout(resolve, 1000));
+		for (let i = 0; i < budgetLineItemInputIds.length; i++) {
+			let budgetLineItemValue = intialFormValue[budgetLineItemInputIds[i].key];
+			let budgetLineItemFieldName = (await dialog.findByTestId(
+				budgetLineItemInputIds[i].id
+			)) as HTMLInputElement;
+			await act(async () => {
+				await fireEvent.change(budgetLineItemFieldName, {
+					target: { value: budgetLineItemValue },
+				});
+			});
+			await expect(budgetLineItemFieldName.value).toBe(budgetLineItemValue);
+		}
+		await act(async () => {
+			let budgetLineItemSaveButton = await dialog.getByTestId("createSaveButton");
+			expect(budgetLineItemSaveButton).toBeEnabled();
+			fireEvent.click(budgetLineItemSaveButton);
+			await wait();
+		});
+		await new Promise((resolve) => setTimeout(resolve, 1000));
+		expect(creationOccured).toBe(true);
```

Comment:
Multiple Expect ? Are we testing multiple functionality in same test suites?

Reply:
yes all of the tests are related to each other, that is why nultiple expect.